guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
<ol><li>Can you spell out FCN? People might not know it at this point.</li><li>Maybe we should soften the tone a bit. Is it always better? If no, we could change it to "Why PSPNet architecture could be an improvement over FCN based segmentation?</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<ol><li>Done</li><li>Moderated the tone.</li></ol>